libsql-sqlite3/src/build.c

```diff
@@ -5563,6 +5571,7 @@ KeyInfo *sqlite3KeyInfoOfIndex(Parse *pParse, Index *pIdx){
   }
   if( pKey ){
     assert( sqlite3KeyInfoIsWriteable(pKey) );
+    pKey->zIndexName = pIdx->zName;
```

Comment:
Does any code in vector search depend on this or is this just an improvement to make zIndexName not null whenever possible?

Reply:
NVM I see that this is a new field.